junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -449,6 +495,25 @@ private static Optional<IndicativeSentencesGeneration> findIndicativeSentencesGe
 			return findAnnotation(testClass, IndicativeSentencesGeneration.class, enclosingInstanceTypes);
 		}
 
+		private static String getSentenceFragment(AnnotatedElement element) {
+			Optional<SentenceFragment> annotation = findAnnotation(element, SentenceFragment.class);
+			if (annotation.isPresent()) {
+				String sentenceFragment = annotation.get().value().trim();
+
+				// TODO [#242] Replace logging with precondition check once we have a proper mechanism for
+				// handling validation exceptions during the TestEngine discovery phase.
+				if (StringUtils.isBlank(sentenceFragment)) {
+					logger.warn(() -> String.format(
+						"Configuration error: @SentenceFragment on [%s] must be declared with a non-empty value.",
```

Comment:
```suggestion
						"Configuration error: @SentenceFragment on [%s] must be declared with a non-blank value.",
```

Reply:
This was copied from `DisplayNameUtils.determineDisplayName(...)`.

So, if we change it here, we should change it in both places for consistency.

Thus, I'll change it in both places.